src/gtk/srchctrl.cpp

```diff
@@ -384,6 +385,43 @@ void wxSearchCtrl::PopupSearchMenu()
 
 #endif // wxUSE_MENUS
 
+wxSize wxSearchCtrl::DoGetBestSize() const
+{
+    int w;
+    GetTextExtent(wxS("0"), &w, nullptr);
+    return DoGetSizeFromTextSize(w * 8);
```

Comment:
This doesn't change much, but why not just

```suggestion
    return DoGetSizeFromTextSize(GetCharWidth() * 8);
```

? Or, if you really prefer to use "0" (even though I don't see why would it be a particularly good choice for a search control), at least

```cpp
    return DoGetSizeFromTextSize(GetTextExtent(wxS("0")).x * 8);
```

Reply:
No reason, just didn't think of it.